### ISSUE TITLE
[Feature request] Determinations Tab - added functionality

### ISSUE THREAD

**Issue by themerekat** [Symbiota/community-board#514]
### Discussed in https://github.com/BioKIC/symbiota-docs/discussions/512

<div type='discussions-op-text'>

<sup>Originally posted by **STHerbariumLSU** May 10, 2024</sup>
Can there be more functionality added to the Determinations Tab?  A few things: 
1. make edits searchable through "date modified".  This is so helpful for checking work.
2. display "family" in the dets. page
3. sync the occ tab and dets tab better so that updates occur when editing either page
4. in the dets tab, don't make determiner or date determined required.  It's not required in the occ. tab.
5. can the Taxon Remarks field not roll over with the determinations?  It makes sense that ID References and ID Remarks rolls over with the Sci Name, but the Taxon field is used more broadly, at least at our institution.  We use it for common names, filed as, checklist identifiers, and general comments.  If others use it more specifically to the sci name, it should probably roll over.  And, it should be added to the Dets tab.
6. I've had trouble with dets not rolling over in the past, but this issue hasn't popped up lately.  Just to keep in mind if work is done here.

Thanks! </div>

**Comment by fbungartz** [comment]
>>1. make edits searchable through "date modified". This is so helpful for checking work.
I agree this would be very helpful, but requires that entering Identification dates and Determiner must remain mandatory.

>>2. display "family" in the dets. page
Not useful for some organisms like lichens, but doesn't hurt either.

>>3. sync the occ tab and dets tab better so that updates occur when editing either page
Yes, I agree!!! It wuld be VERY helpful to make sure that ANY edits on the Occurrences will automatically also create an annotation of the Determination Tab.

>>4. in the dets tab, don't make determiner or date determined required. It's not required in the occ. tab.
I would strongly encourage that this should be mandatory also on the Occurrence Tab (see comment for #1).

>>5. can the Taxon Remarks field not roll over with the determinations? It makes sense that ID References and ID Remarks rolls over with the Sci Name, but the Taxon field is used more broadly, at least at our institution. We use it for common names, filed as, checklist identifiers, and general comments. If others use it more specifically to the sci name, it should probably roll over. And, it should be added to the Dets tab.

I do not understand nor do I agree with this request. On the Determinations Tab the Taxon Remarks is currently MISSING. The Taxon Remarks are present only on the main Occurrence Tab. 
Perhaps, there is some misunderstanding here, because the fields are named differently:
Occurrence Tab                     Determination Tab
ID References               =      References
Identification Remarks  =      Notes
Taxon Remarks             =      not available

I think both ID References (= References) and Identification Remarks (= Notes) must be updated on the Occurrence Tab (as it currently is correctly implemented). The References refer to the literature used for identifying a specimen, the Identification Remarks should be used for descriptive comments referring to the diagnostic characters used for the identification of the specimen.

I would much prefer, if also the third field Taxon Remarks was available and roll over on the Determinations Tab, because that is really a comment field referring to the taxonomy, e.g., when an author makes a comment about the taxonomy of the species; that field would also then be useful for explaining nomenclatural adjustments.

So, my request would be exactly the opposite, i.e., to ADD the Taxon Remarks field to the Determination Tab (where it is currently missing).
And PLEASE do not remove any current rollover of the  ID References (= References) and Identification Remarks (= Notes) to the current ID on the Occurrences. At least this IS working and that functionality must NOT be removed.

>>6. 
I've had trouble with dets not rolling over in the past, but this issue hasn't popped up lately. Just to keep in mind if work is done here.

It remains an issue, especially sometimes generating erroneous duplicates of annotations. This is why I would strongly recommend that the very first annotation is automatically generated on the Determinations Tab as soon as any ID is entered on the Occurrence tab, and not only when an annotation of the current ID is entered there. The Determination History should always show all determinations, including the first one, even if there is only that very first one. Ideally, if the identification gets overwritten on the occurrence tab, a new annotation should automatically be added on the determination tab.  Only this workflow would assure that no info is lost.

**Comment by fbungartz** [comment]
PS: the field size for the Notes (Identification Remarks) is rather small:
![image](https://github.com/BioKIC/symbiota-docs/assets/42786860/2c340aa5-cc9a-4c21-971a-197aa6cd47f6)
![image](https://github.com/BioKIC/symbiota-docs/assets/42786860/8ad9ffa1-a6d6-48b3-b20c-59225452b460)

**Comment by fziemme** [comment]
Thank for adding this, but I cannot access the link.
